### PR TITLE
[#2342] Fixed issues with share-editor and header being out of sync

### PR DESCRIPTION
**Pull request by dseif**

### CODE REVIEW

File: src/ui/header.js [hunk 1]
```diff
@@ -103,6 +103,7 @@ define([ "dialog/dialog", "util/lang", "ui/user-data", "ui/widget/tooltip" ],
      * @returns boolean value representing whether or not the current project name is valid
      */
     function checkProjectName() {
+      console.log( _projectTitlePlaceHolderText, butter.project.name );
```
Comment (cadecairos) [inline src/ui/header.js]: remove plz :D

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/0a716c09864fbb9/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/6dea28e6a94d26f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/0a716c09864fbb9/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/6dea28e6a94d26f/output.txt

Total script time: 3.00 mins

**Pull request closed without merging**